client/packages/invoices/src/StockOut/useAllocationContext.ts

```diff
@@ -257,24 +257,24 @@ export const useAllocationContext = create<AllocationContext>((set, get) => ({
         ? result.remainingQuantity
         : 0;
 
+    // Note that a placeholder is always considered to be pack size 1, 1 dose per unit
+    // So if issuing in larger pack sizes, we need to adjust the placeholder
+    const placeholderInUnits =
+      allocateIn.type === AllocateInType.Packs
+        ? stillToAllocate * allocateIn.packSize
+        : stillToAllocate;
+
     const alerts = getAllocationAlerts(
       quantity,
       allocatedQuantity,
-      stillToAllocate,
+      placeholderInUnits,
```

Comment:
last unrelated fix lol - placeholder is in units, so warning should show in units too 😅